api/auth/index.test.js

```diff
@@ -169,17 +183,54 @@ tap.test('authentication setup', async authTest => {
     getTest.ok(res.end.calledOnce, 'response is ended one time');
   });
 
-  authTest.test('POST login endpoint behaves as expected', async postTest => {
-    authSetup(app, passport, strategies);
-    const post = app.post.args[0][2];
+  authTest.test('POST nonce endpoint behaves as expected', async nonceTests => {
+    nonceTests.test('sends a 400 error if there is no body', async test => {
+      authSetup(app, { auth, passport, session, strategies });
+      const post = app.post.args.find(a => a[0] === '/auth/login/nonce')[1];
 
-    const res = {
-      send: sinon.stub(),
-      status: sinon.stub(),
-      end: sinon.spy()
-    };
-    res.send.returns(res);
-    res.status.returns(res);
+      post({}, res);
+
+      test.ok(res.status.calledOnce, 'an HTTP status is set once');
+      test.ok(res.status.calledWith(400), 'sets a 200 HTTP status');
```

Comment:
wait, the status  returns 200 for a 400?

Reply:
Doh.  Will fix the typo.